pygfx/renderers/wgpu/shaders/backgroundshader.py

```diff
@@ -171,12 +171,24 @@ def code_fragment(self):
             // So we fool the blender into thinking this fragment is opaque, even if its not.
             var out = get_fragment_output(varyings.position.z, vec4<f32>(out_color.rgb, 1.0));
             $$ if write_pick
-            out.pick = (
-                // For a background, it doesn't make sense to report back
-                // anything other than the background id.
-                pick_pack(u32(u_wobject.id), 20)
-                // Maybe this can be different for for textured backgrounds?
-            );
+                $$ if texture_dim == 'cube'
+                    out.pick = (
+                        // For a background, it doesn't make sense to report back
+                        // anything other than the background id.
+                        pick_pack(u32(u_wobject.id), 20) +
+                        pick_pack(u32(varyings.texcoord.x * 16383.0), 14) +
+                        pick_pack(u32(varyings.texcoord.y * 16383.0), 14) +
+                        pick_pack(u32(varyings.texcoord.z * 16383.0), 14)
```

Comment:
Hmmm... this doesn't seem to be working..

Reply:
The `z` is (semantically) an integer, I think.

Reply:
hmmm, yeah that didn't help, with my debugging output  (enabled on this branch)

```
python examples/feature_demo/skybox.py
```

just prints clamped values most of the time. Sometimes..... it prints non-clamped values.